.gitignore

```diff
@@ -2,3 +2,6 @@
 npm-debug.log
 package-lock.json
 yarn.lock
+.idea
+eslint-report.html
+/test
```

Comment:
Please only ignore `/test/fixtures`. Also, end the file with a new line; GitHub even warns about it with a red stop symbol here at the end.